flang/lib/Parser/prescan.cpp

```diff
@@ -1186,6 +1186,8 @@ Prescanner::IsFixedFormCompilerDirectiveLine(const char *start) const {
         break;
       }
       *sp++ = ToLowerCaseLetter(*p);
+    } else {
```

Comment:
This is just a comment and no action is required.

A hypothetical line starting with `!$fp +` would not be considered a continuation of a previous line starting with `!$fp ` line as we check for `column == 6` below.

However, given that we do not have sentinels of length 2 or 3 (only 1 or 4, and length 1 is already handled in a special way) I believe this doesn't need solving now (I don't think one could write a test for that case currently).